### ISSUE TITLE
@mention autocompletion

### ISSUE THREAD

**Issue by dmzaytsev** [yegor256/netbout#606]
I believe netbout needs `@mention` autocompletion like Github.
When you type an @ and you should see a list of people on the boat. Continue typing to filter the list and hit enter or tab to complete the name. 

updated 6/10/2015:
you might to use e.g. this autocompletion implementation https://github.com/podio/jquery-mentions-input but feel free to propose another. I spent 3 mins to google it....

**Comment by dmzaytsev** [comment]
@karato it's a valid bug

**Comment by dmzaytsev** [comment]
@karato please set millestone to 3.0

**Comment by yegor256** [comment]
@dmzaytsev yes, that would be awesome

**Comment by dmzaytsev** [comment]
@karato this issue is postponed

**Comment by karato** [comment]
> @karato it's a valid bug

@dmzaytsev tagged this issue with "bug"

**Comment by karato** [comment]
@dmzaytsev I set milestone here to `3.1`, let me know if it is wrong

**Comment by karato** [comment]
> @karato please set milestone to 3.0

@dmzaytsev I attached this issue to milestone `3.0`

**Comment by karato** [comment]
> @karato this issue is postponed

@dmzaytsev sure, thanks, I added "postponed" tag to it

**Comment by karato** [comment]
@dmzaytsev thank you for the ticket reported, I topped your account for 15 mins, transaction `57699831`

**Comment by karato** [comment]
@ekondrashev this is your task

**Comment by ekondrashev** [comment]
@karato This one is blocked by #623 , we need to decide which lib to use to implement autocompletion.

**Comment by karato** [comment]
> @karato please set milestone to 3.0

@dmzaytsev I attached this issue to milestone `3.0`

**Comment by karato** [comment]
> @karato this issue is postponed

@dmzaytsev sure, thanks, I added "postponed" tag to it

**Comment by karato** [comment]
> @karato this issue is postponed

@dmzaytsev OK, I will try to assign someone else

**Comment by karato** [comment]
@antonini could you please pick this up? This [article](http://www.xdsd.org/2014/04/17/how-xdsd-is-different.html) explains how we work. Any technical questions you may ask right here. The budget here is **30 mins**, which is exactly how much time will be paid for, when the task is [completed](http://www.xdsd.org/2014/04/15/definition-of-done.html)

**Comment by dmzaytsev** [comment]
@karato why did you change the ticket performer?

**Comment by yegor256** [comment]
@karato it's postponed, please remove the assignee

**Comment by karato** [comment]
> @karato it's postponed, please remove the assignee

@yegor256 sure, thanks, I added "postponed" tag to it

**Comment by karato** [comment]
> @karato it's postponed, please remove the assignee

@yegor256 OK, I will try to assign someone else

**Comment by karato** [comment]
@erimerturk the task is for you now, follow [these guidelines](http://www.xdsd.org/2014/04/17/how-xdsd-is-different.html). Don't hesitate to ask any technical questions right here; The cost of this task is **30 mins** (this is exactly how much will be paid, not less not more), when the task is [done](http://www.xdsd.org/2014/04/15/definition-of-done.html)

**Comment by erimerturk** [comment]
@yegor256 @karato  is this task postponed or not ? I am confused because of the comments. thank you.

**Comment by yegor256** [comment]
@karato it is postponed indeed, don't assign it to anyone pls

**Comment by karato** [comment]
> @karato it is postponed indeed, don't assign it to anyone pls

@yegor256 right, I added "postponed" tag

**Comment by karato** [comment]
> @karato it is postponed indeed, don't assign it to anyone pls

@yegor256 no problem, I will try to find somebody else

**Comment by dmzaytsev** [comment]
@karato this task not postponed any more

**Comment by karato** [comment]
> @karato this task not postponed any more

@dmzaytsev thanks, I removed the "postponed" tag

**Comment by karato** [comment]
@mbarbieri the task is for you now, follow [these guidelines](http://www.xdsd.org/2014/04/17/how-xdsd-is-different.html). Don't hesitate to ask any technical questions right here

The cost of this task is **30 mins** (this is exactly how much will be paid, not less not more), when the task is [done](http://www.xdsd.org/2014/04/15/definition-of-done.html)

**Comment by dmzaytsev** [comment]
@mbarbieri  what's the status here?

**Comment by mbarbieri** [comment]
@dmzaytsev  working on it

**Comment by dmzaytsev** [comment]
@mbarbieri please keep in mind http://www.teamed.io/policy.html#16

**Comment by mbarbieri** [comment]
@dmzaytsev @yegor256 What is the correct way to include external jQuery plugins?
If I copy the js file in js folder it will be checked by qulice, the plugin isn't available in a cdn and directly link to github doesn't sound good...

**Comment by dmzaytsev** [comment]
@mbarbieri you might copy the js file into js folder. If qulice will be complain on it, just exclude the js file from the checks. see http://www.qulice.com/qulice-maven-plugin/example-exclude.html

**Comment by dmzaytsev** [comment]
@mbarbieri I just would remind, you should inform the PM about the delay and ask more time. It's your responsibility as the ticket performer. You will be fined in 7 days and a task will be reassigned in 10 days on the commercial projects

**Comment by yegor256** [comment]
@mbarbieri @dmzaytsev having a third-party code in our repo is a **bad** practice. We should find a way how to use their code without copy-pasting

**Comment by dmzaytsev** [comment]
@mbarbieri ask for more time ASAP  if you don't want loose this task

> dmzaytsev commented 5 days ago
> @mbarbieri I just would remind, you should inform the PM about the delay and ask more time. It's your responsibility as the ticket performer. You will be fined in 7 days and a task will be reassigned in 10 days on the commercial projects

**Comment by mbarbieri** [comment]
@karato can I have more time to work on this?

**Comment by karato** [comment]
> @karato can I have more time to work on this? 

@mbarbieri OK, no problem, take your time

**Comment by karato** [comment]
@dmzaytsev once `606-457eb15d`/#723 puzzle is resolved (later, in another ticket), this ticket will be fully complete

**Comment by karato** [comment]
@rultor rultor closed it (automatically), @dmzaytsev please take a look

**Comment by dmzaytsev** [comment]
@mbarbieri thanks!

**Comment by karato** [comment]
@ypshenychka please, review this task for compliance with our [quality rules](http://www.teamed.io/qa.html)

**Comment by ypshenychka** [comment]
@mbarbieri 
According to our [QA Rules](http://www.teamed.io/qa.html):
>Some of ticket messages must mention all pull requests where the problem was actually fixed.

Please mention in any of your messages link to corresponding PR.

**Comment by mbarbieri** [comment]
@ypshenychka done

**Comment by ypshenychka** [comment]
@mbarbieri Thank you.

**Comment by ypshenychka** [comment]
@karato Quality is acceptable.

**Comment by karato** [comment]
> @karato Quality is acceptable.

@ypshenychka thanks a lot, next time everybody should try to make it better

**Comment by karato** [comment]
@mbarbieri I added 10 mins to @ypshenychka (for QA review) in transaction 60283990... thank you, added **30 mins** to your acc, payment num is `AP-5B4161993B003974A`, 401 hours and 16 mins spent in total to complete... +30 to your rating, your total score is [-7](http://www.netbout.com/b/35357?open=rating)

**Comment by antonini** [comment]
@dmzaytsev Puzzle #723 was merged. Can you check it?

**Comment by dmzaytsev** [comment]
@antonini it works
you can see on snapshot.netbout.com

**Comment by antonini** [comment]
@dmzaytsev Yep.. I just checked. I have one doubt. Who closes puzzle? You (as reporter of this issue) or PM (as reporter of puzzle issue)?

**Comment by dmzaytsev** [comment]
@antonini @karato will close the puzzle soon

**Comment by karato** [comment]
@dmzaytsev the last puzzle `606-457eb15d`/#723 originated from here solved
